workshops/guided-projects/guided_project_1.ipynb

```diff
@@ -4,14 +4,24 @@
    "cell_type": "markdown",
```

Comment:
<p>Nit: Really like the debugging callouts and instructions. Maybe consider <strong>Tip: </strong>or  <strong>Debugging tip:</strong> moving this up under note so this important note stands out more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Done!</p>